NewDawn/NewDawn/Models/UserModels.swift

```diff
@@ -77,7 +77,7 @@ let USER_DUMMY_DATA_2: NSDictionary = [
     ]
 ]
 
-let USER_DUMMY_DATA = [USER_DUMMY_DATA_1, USER_DUMMY_DATA_2]
+var USER_DUMMY_DATA = [USER_DUMMY_DATA_1, USER_DUMMY_DATA_2]
```

Comment:
Eventually this should be fetched from backend on daily refresh, and stored into local storage either as json strings or as UserProfile objects. Then main page will only read it from local storage.